sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -418,7 +379,7 @@ func (m *marshaller) addMultiEdge(edge NamedEdge) ([]string, error) {
 
 				si[fmt.Sprintf("i%v", i)] = &pipepb.SideInput{
 					AccessPattern: &pipepb.FunctionSpec{
-						Urn: URNMultimapSideInput,
+						Urn: URNIterableSideInput,
 					},
 					ViewFn: &pipepb.FunctionSpec{
 						Urn: "foo",
```

Comment:
ViewFns are not currently implemented in the Go SDK but the proto requires the field to be non-empty. So we put a dummy placeholder URN in place for now. The WindowMappingFn field had the same set-up until I implemented the basic window mapping functionality we use now. 

"Required" fields in protos are a legacy thing that are discouraged/outright unsupported now since they're a headache to deal with, generally.

Reply:
In this case, it's "required" by some runner implementations to be non-empty, if not actually required by explicit proto semantics.